tzutalin/labelImg
# Improve project structure

This changes the project structure to be compliant with both Python conventions and commonly used tools. I used the cookiecutter-pypackage project to generate the files.

Among the features that it brings:
* Sphinx documentation
* Deployment to PyPi
* Makefile to ease common tasks
* Tox and PyTest to create multi-platform unit tests
* Requirements files to install dependencies

There would still be a bit of work to continue this:
* Make the installer generators compatible
* Create the PyPi repository and configure travis (see the travis_pypi_setup.py script)
* Complete HISTORY files for the description of the release
* Complete module documentation
